schemas/context/profile.schema.json

```diff
@@ -86,6 +91,24 @@
           "description":
             "The unit ID within the organization owning the profile. This ID can be used to reference the organization details maintained in another dataset."
         },
+        "xdm:preferredLanguage": {
+          "title": "Preferred Language",
+          "type": "string",
+          "description":
+            "Preferred Language"
+        },
+        "xdm:thumbnail": {
```

Comment:
thumbnail is something completely different.  This needs a name like "profileLink" to correct describe it.  (also it doesn't belong in the top level namespace)

Reply:
Changed the name to "profilePictureLink" in latest commit.